app/assets/main/react_components/footprint_result/SignUpContainer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import MembershipTypeChooser from './MembershipTypeChooser.jsx';
+import RefferalCode from './ReferralCode.jsx';
+import SignUpMotivationText from './SignUpMotivationText.jsx';
+import SignUpTitle from './SignUpTitle.jsx';
+
+/**
+ * React container for Sign up components
+ */
+const SignUpContainer = ({ registrationsText }) => {
+
+    return (
+        <div className="relative pb-1">
+            <div className="space-y-6">
+                <div className="callout max-w-md mx-auto">
+                    <a id="sign-up" className="absolute -mt-32"></a>
+                    <div className="space-y-3" data-controller="registrations--price registrations--referral-code">
+                        <SignUpTitle
+                            signUpTitleText={registrationsText.sign_up_heading_test_2}
+                        />
+                        <SignUpMotivationText
+                            signUpMotivationText={registrationsText.sign_up_description}                            
+                        />
```

Comment:
Detta är så himla oviktigt, men kanske matcha "sign up" istället för "registration" i registrationsText, för att det ska bli tydligare? Gör som du vill

Reply:
"Registrations" används i RoR som term, därför har det hängt kvar som props-benämning. Däremot anser jag att signup är en mer beskrivande term för just det stället man skriver in sig på. Jag behåller det gärna, men om vi ska vara enhetliga med backend kanske till och med Registrations passar bättre, men är mycket att ändra på. Vad tycker ni? @e-ry @mabackstr

Reply:
@teolenn @e-ry Känns bra att ha att åtminstone när texterna skickas som props till ReactContainer från RoR så heter de som sin YML-fil. Då är det lättare att veta vilken fil som skickats. Sen när värdena skickas vidare till children från ReactContainer så kan man ju ändra om det blir mer intuitivt. Så till SignUpContainer kan man skicka in: 
`<SignUpContainer signUpText={JSON.parse(registrationsText)} />`  om det föredras, jag har ingen preferens

Reply:
Yes det kan jag gå med på! 😊 Då sker det alltså en gränsdragning i ResultContainer

Reply:
@e-ry Funkar det för dig?

Reply:
Funkar alldeles utmärkt för mig!